package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-tracking",
-  "version": "7.2.1",
+  "version": "7.2.2",
```

Comment:
Please revert this change, version bumps are part of the release process

Reply:
My bad, fixed with 9c47128